src/conf.py

```diff
@@ -57,17 +58,17 @@
 
 # General information about the project.
 project = u'conda-forge'
-copyright = u'2016-2019, conda-forge'
```

Comment:
Does it make sense to also generate the current year automatically?

Reply:
good idea

Reply:
done.